benchmarks/votable.py

```diff
@@ -177,6 +187,48 @@ def time_booleans_binary2(self):
         parse(io.BytesIO(self.binary2_data))
 
 
+class TimeVOTableBitArrayOptimization:
+    """Benchmark BitArray columns in Binary/Binary2 VOTables."""
+
+    def setup(self):
+        table = Table(
+            {
+                "ra": ra_data[:LARGE_SIZE],
+                "dec": dec_data[:LARGE_SIZE],
+                "mag": mag_data[:LARGE_SIZE],
+                "detected": rng.integers(0, 2, LARGE_SIZE).astype(bool),
+                "saturated": rng.integers(0, 2, LARGE_SIZE).astype(bool),
+                "edge_pixel": rng.integers(0, 2, LARGE_SIZE).astype(bool),
+                "cosmic_ray": rng.integers(0, 2, LARGE_SIZE).astype(bool),
+            }
+        )
+
+        self.binary_bitarray_8_data = create_votable_bytes(
+            table, "binary", "8")
```

Comment:
There's some new code here that hasn't been formatted with Ruff. New code should be formatted with Ruff so that when Ruff is adopted in this repository then the patch to update the formatting would be smaller.

Reply:
This isn't core library. We never even discussed PEP 8 here, much less ruff. So I wouldn't suddenly enforce that now as a rule that would block merge here.